Corgibytes.Freshli.Cli/CommandRunners/Cache/PrepareCacheActivity.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using Corgibytes.Freshli.Cli.DataModel;
+using Corgibytes.Freshli.Cli.Extensions;
+using Corgibytes.Freshli.Cli.Functionality;
+using Corgibytes.Freshli.Cli.Functionality.Engine;
+using Corgibytes.Freshli.Cli.Resources;
+
+namespace Corgibytes.Freshli.Cli.CommandRunners.Cache;
+
+public class PrepareCacheActivity : IApplicationActivityEngine, IApplicationActivity
+{
+    public PrepareCacheActivity(string cacheDirectory, string repositoryUrl = "", string? repositoryBranch = null,
+        string historyInterval = "")
+    {
+        CacheDirectory = cacheDirectory;
+        RepositoryUrl = repositoryUrl;
+        RepositoryBranch = repositoryBranch;
+        HistoryInterval = historyInterval;
+    }
+
+    public string RepositoryUrl { get; init; }
+
+    public string? RepositoryBranch { get; init; }
+
+    // TODO: Research how to use a value class here instead of a string
+    public string HistoryInterval { get; init; }
+
+    public string CacheDirectory { get; init; }
+
+    public void Handle(IApplicationEventEngine eventClient)
+    {
+        var cacheManager = new CacheManager();
+        Console.Out.WriteLine(CliOutput.CachePrepareCommandRunner_Run_Preparing_cache, CacheDirectory);
```

Comment:
We can leave this as is for now, but we should limit writing to the console from activity handler methods. I'll create an issue to refactor this.